orchagent/srv6orch.cpp

```diff
@@ -484,7 +484,11 @@ bool Srv6Orch::createUpdateMysidEntry(string my_sid_string, const string dt_vrf,
     {
         sai_object_id_t dt_vrf_id;
         SWSS_LOG_INFO("DT VRF name %s", dt_vrf.c_str());
-        if (m_vrfOrch->isVRFexists(dt_vrf))
+        if (dt_vrf == "default")
```

Comment:
@link19430 could you please update test_srv6.py with one mysid entry with default VRF?